src/styles/blocks/product-card.scss

```
@@ -0,0 +1,94 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &-image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+    margin-bottom: 40px;
+    background-size: 100% 100%;
+  }
+
+  &-text {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
+    color: $main-text-color;
+    margin-bottom: 4px;
+  }
+
+  &-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+    margin-bottom: 16px;
+  }
+
+  &-rating {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &-menu {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    margin-bottom: 16px;
+
+    .menu-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $second-text-color;
+    }
+
+    .menu-count {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &-button {
+    width: 166px;
```

let's use max-width instead:
```suggestion
    max-width: 166px;
    width: 100%;
```